CurrentTimeApp.MauiBlazor/packages.lock.json

```diff
@@ -227,45 +227,45 @@
       },
       "Microsoft.AspNetCore.Authorization": {
```

Comment:
An error occurred with the MAUI packages.lock.json. Some additional transitive dependencies need to be updated:

```
D:\a\CurrentTimeApp\CurrentTimeApp\CurrentTimeApp.MauiBlazor\CurrentTimeApp.MauiBlazor.csproj : error NU1004: The package reference Microsoft.Maui.Graphics version has changed from [7.0.52, ) to [7.0.59, ).
D:\a\CurrentTimeApp\CurrentTimeApp\CurrentTimeApp.MauiBlazor\CurrentTimeApp.MauiBlazor.csproj : error NU1004: The package reference Microsoft.Maui.Graphics version has changed from [7.0.52, ) to [7.0.59, ).
D:\a\CurrentTimeApp\CurrentTimeApp\CurrentTimeApp.MauiBlazor\CurrentTimeApp.MauiBlazor.csproj : error NU1004: The package reference Microsoft.Maui.Graphics version has changed from [7.0.52, ) to [7.0.59, ).
D:\a\CurrentTimeApp\CurrentTimeApp\CurrentTimeApp.MauiBlazor\CurrentTimeApp.MauiBlazor.csproj : error NU1004: The package reference Microsoft.Maui.Graphics version has changed from [7.0.52, ) to [7.0.59, ).The packages lock file is inconsistent with the project dependencies so restore can't be run in locked mode. Disable the RestoreLockedMode MSBuild property or pass an explicit --force-evaluate option to run restore to update the lock file.
  Failed to restore D:\a\CurrentTimeApp\CurrentTimeApp\CurrentTimeApp.MauiBlazor\CurrentTimeApp.MauiBlazor.csproj (in 245 ms).
  Restored D:\a\CurrentTimeApp\CurrentTimeApp\CurrentTimeApp.Components\CurrentTimeApp.Components.csproj (in 4.37 sec).
```